lib/spack/llnl/util/tty/pty.py

```diff
@@ -156,20 +156,20 @@ def wait_running(self):
 
 
 class PseudoShell(object):
-    """Sets up master and child processes with a PTY.
+    """Sets up controller and child processes with a PTY.
 
     You can create a ``PseudoShell`` if you want to test how some
     function responds to terminal input.  This is a pseudo-shell from a
-    job control perspective; ``master_function`` and ``child_function``
```

Comment:
If we have "child_function" shouldn't the opposite be "parent_function"?

Reply:
I guess I look at it as "the controller of the child process".  Parent was confusing because there are actually 3 processes involved.  The controller is the child of the main process and the parent of the children.

Reply:
I changed this to controller/minion per discussion